docs/admin.rst

```diff
@@ -134,10 +134,8 @@ characters that are highlighted in yellow.
 Inside the Google Authenticator app, a new entry for this account will
 appear on the main screen, with a six digit number that recycles to a
 new number every thirty seconds. Enter the six digit number under
-“Verification code” at the bottom of the SecureDrop screen here, and hit
-enter:
-
-|Verify Google Authenticator works|
+“Verification code” at the bottom of the screen here, and hit
+enter.
```

Comment:
The former screenshot was the same as the above and does not need repeating. It was slightly different in the previous version with a red arrow indicating the bottom of the screen but this is not something the automatic screenshot can provide and this is only marginally useful. I agree scarifying it is acceptable.